spec/requests/calculators_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe "Calculators", type: :request do
+  describe "GET /index" do
+    it "returns http success" do
+      get "/calculators/index"
```

Comment:
1. юзай path хелпери
2. be_successful
```suggestion
      get "/calculators/index"

```